src/examples/src/widgets/menu/FetchedResource.tsx

```diff
@@ -0,0 +1,61 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
+import List from '@dojo/widgets/list';
+import icache from '@dojo/framework/core/middleware/icache';
+import { createResource, createTransformer, DataTemplate } from '@dojo/widgets/common/resource';
+
+const fetcher = async (options: any) => {
```

Comment:
Might as well make this non remote for the examples (as my server is slow and if it breaks the examples break)

Reply:
no worries, can do